src/natcap/invest/recreation/recmodel_client.py

```diff
@@ -1502,6 +1505,33 @@ def error_handler(err_level, err_no, err_msg):
             "projection of the base vector")
 
 
+def _validate_predictor_types(table_path):
+    """Validate the type values in a predictor table.
+
+    Args:
+        table_path (string): path to a csv table that has at least
+            the field 'type' 
+
+    Returns:
+        None
+
+    Raises:
+        ValueError if any value in the ``type`` column does not match a valid
+        type, ignoring leading/trailing whitespace.
+    """
+    df = utils.read_csv_to_dataframe(table_path, to_lower=True)
+    type_list = df['type'].tolist()
+    valid_types = ['raster_mean', 'raster_sum', 'point_count', 
```

Comment:
I noticed that under the `execute` docstring the `predictor_table_path` doesn't include the `polygon_percent_coverage`, but I do see it in the User's Guide and that it is used in the model. Could you add it to the `execute` docstring?

Reply:
Oh, good catch!